python/ray/serve/tests/test_telemetry.py

```diff
@@ -42,6 +43,17 @@ def check_ray_started():
     return requests.get("http://localhost:52365/api/ray/version").status_code == 200
 
 
+def check_telemetry_recorded(storage_handle, key, expected_value):
+    report = ray.get(storage_handle.get_report.remote())
+    assert report["extra_usage_tags"][key] == expected_value
+    return True
+
+
+def check_telemetry_not_recorded(storage_handle, key):
+    report = ray.get(storage_handle.get_report.remote())
+    assert key not in report["extra_usage_tags"]
```

Comment:
you can use the `ServeUsageKey.get_value_from_report` method for this (check `is None`)